src/frontends/tensorflow_lite/src/op_table.cpp

```diff
@@ -28,7 +28,7 @@ using namespace ov::frontend::tensorflow::op;
         auto inputs = node.get_inputs();                                                                           \
         ov::frontend::tensorflow_lite::dequantize_inputs(inputs);                                                  \
         auto context = frontend::tensorflow_lite::NodeContext(decoder, inputs);                                    \
-        return func(context);                                                                                      \
+        return get_indexed_outputs(func(context));                                                                 \
```

Comment:
why line 23 is not changed?

Reply:
23 can be changed also, but it is not used for common translators from tensorflow_common, so I decided that it is not needed